packages/docs/src/locales/en/en.json

```diff
@@ -3354,6 +3354,10 @@
         "title": "Sizes",
         "text": "You can set different sizes."
       },
+      "grow": {
+        "title": "Grow",
+        "text": "Makes button group grow to the width of its container."
```

Comment:
Missing translation in the russian, it's not necessary for now, because we have no translations in some pages too. I'll move it to the new issue, where we can check all pages for missing translations.

Reply:
Yeah. This is a thing I don't care for now.